src/mediarithmics/api/core/plugin/ValueInterface.ts

```diff
@@ -4,6 +4,14 @@ export interface AssetFilePropertyResource {
   file_path?: string;
 }
 
+export interface AssetFolderPropertyResource {
```

Comment:
I think that we deprecated some fields on `AssetFilePropertyResource` the other day.

Can you add the new fields on the `AssetFilePropertyResource` interface (and keep the Deprecated ones)